x-pack/test/functional/services/ml/job_table.ts

```diff
@@ -48,8 +54,25 @@ export function MachineLearningJobTableProvider(
   const headerPage = getPageObject('header');
   const testSubjects = getService('testSubjects');
   const retry = getService('retry');
+  const find = getService('find');
 
   return new (class MlJobTable {
+    public async assertJobsInTable(expectedJobIds: string[]) {
+      for await (const expectedId of expectedJobIds) {
+        const visibleText = await testSubjects.getVisibleText('mlJobListColumnId');
+        expect(visibleText).to.be(expectedId);
+      }
+    }
+    public async filterByState(quickFilterButton: QuickFilterButtonTypes): Promise<void> {
+      await find.clickByCssSelector(
+        `[data-test-subj="mlJobListSearchBar"] span[data-text="${quickFilterButton}"]`
+      );
+      const ariaPressed: boolean = await find.existsByCssSelector(
+        `[data-test-subj="mlJobListSearchBar"] button[aria-pressed="true"] span[data-text="${quickFilterButton}"]`
+      );
+      expect(ariaPressed).to.be(true);
```

Comment:
Would be good to add an assertion message (otherwise a failure message would be something like `expected false to be true` which isn't very helpful for debugging).

Reply:
Oh so right boss man.

Reply:
https://github.com/elastic/kibana/pull/179321/commits/f9fdc495f4b8a6c402567294dc91621c599c065a